go/kubebuilder/memcached-operator/README.md

```diff
@@ -69,6 +89,16 @@ NAME                                                               DESIRED   CUR
 replicaset.apps/memcached-operator-controller-manager-864f7c75d4   1         1         1       118s
 ```
 
+### Verifying the validating webhook
+
+The following command attempts to increase the CR's `spec.size` to an even number. It should throw an error, like that shown below, as the validating webhook does not allow an even `spec.size`.
```

Comment:
```suggestion
The following command attempts to increase the CR's `spec.size` to an even number. It should throw an error like that shown below, as the validating webhook does not allow an even `spec.size`.
```

Reply:
@estroz Is it ok if I address these nits in another PR? Due to the revert that happened in earlier commit for `ansible/Memcahced` folders, my Mac is not allowing me to move forward with any kind of rebases or commits. I am concerned that this feature branch will be effected, if I delete my fork.

Reply:
You can commit them via Github (see the `Commit suggestion` button).